tests/__init__.py

```diff
@@ -23,4 +23,4 @@
 
 from .api import *
 from .core import *
-from .models import *
+from .test_models import *
```

Comment:
If it is called test_models.py it doesn't need to be imported explicitlty anymore (which is better. This was a kludge for not following python conventions).

Reply:
yes indeed, this was auto generated by my idea, can remove that